Marsiple/App/New Group/AlbumsViewController.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  AlbumsViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 18/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import SnapKit
+
+class AlbumsViewController: UIViewController {
+    private let albums = [Album](repeating: Album(images: [#imageLiteral(resourceName: "Image1"), #imageLiteral(resourceName: "Image2"), #imageLiteral(resourceName: "Image3"), #imageLiteral(resourceName: "Image4"), #imageLiteral(resourceName: "Image5"), #imageLiteral(resourceName: "Image1"), #imageLiteral(resourceName: "Image2"), #imageLiteral(resourceName: "Image3"), #imageLiteral(resourceName: "Image4")]), count: 10)
```

Comment:
Lets a `TODO:` or `NOTE:` here, that this is just placeholder code, to be replaced with a concrete implementation in the future.